R/TuningInstance.R

```diff
@@ -274,6 +274,7 @@ TuningInstance = R6Class("TuningInstance",
         a = bmr$aggregate(measures = self$measures, ids = FALSE)[, mids, with = FALSE]
         lg$info("Result:")
         lg$info(capture.output(print(cbind(dt, a), class = FALSE, row.names = FALSE, print.keys = FALSE)))
+        lg$info(paste("Number of configuration evaluations:", self$n_evals))
```

Comment:
Is the wording here clear enough? Maybe

```suggestion
        lg$info(paste("Iteration #:", self$n_evals))
```

sufficient?